File: CourseOOP/Python/card.py
from numpy import datetime64
from payment import Payment

class Card(Payment):
    number = int
    CVV_number = int
    date = datetime64

    def __init__(self, id, number, CVV_number, date):
        super().__init__(id)
        self.number = number
        self.CVV_number = CVV_number
        self.date = date

File: CourseOOP/Java/Card.java
package Java;

import java.util.Date;

public class Card extends payment{
    Integer number;
    Integer CVV_number;
    Date date;

    public Card(Integer id, Integer number, Integer CVV_number, Date date){
        super(id);
        this.number = number;
        this.CVV_number = CVV_number;
        this.date = date;
    }
}
